synthesizer/msynthesizer.cpp

```diff
@@ -52,7 +52,10 @@ void MasterSynthesizer::init()
             else
                   e.unknown();
             }
-      setState(state);
+      if (!setState(state)) {
+            f.remove();
```

Comment:
how about not removing, but still set default state? So in case the missing soundfont gets reinstalled, the old config starts working again?